### DISCUSSION TITLE
Old Menu

### DISCUSSION THREAD

**Discussion by vardash** [Comfy-Org/ComfyUI#4728] [General]
Hello everyone,

I wanted to ask if it is possible to get the old menu back where everything was at the top.

I find the queue together with the history confusing, furthermore it no longer seems to be possible to cancel individual batch jobs, only the current one, which in my opinion makes little sense if I want to cancel the jobs furthest down, for example

**Comment by ltdrdata** [comment]
1. You can disable new UI on settings.
![image](https://github.com/user-attachments/assets/bc4fe10b-3597-47f2-aadd-7d07b941d670)


2. You can cancel specific queued item on side bar's history preview. (context menu)
![image](https://github.com/user-attachments/assets/cae3f960-d3f8-4c3f-b9e3-7b7ca99ef7d4)

**Comment by jau123** [comment]
@ltdrdata In the new interface, how can I enter the settings page?

**Reply by ltdrdata** [reply]
You can find the button on the left-bottom corner.
